AzureMonitorAgent/HandlerManifest.json

```diff
@@ -17,21 +17,26 @@
       "services": [
         {
           "name": "azuremonitoragent"
+          "cpuQuotaPercentage": 250
         },
         {
           "name": "azuremonitoragentmgr"
```

Comment:
azuremonitoragentmgr cgroup may have any number of mdsd tenants running under it in 1p case. as such, we cannot reasonably set a limit